### PR TITLE
[16.0][MIG] stock_account_valuation_report: Migration to 16.0

### PR DESCRIPTION
**Pull request by RicardCForgeFlow**
Migration from 15.0 to 16.0

### CODE REVIEW

**Review by CasVissers-360ERP — APPROVED** [review]
We have this running for some time now.

**Review by astirpe — APPROVED** [review]

File: stock_account_valuation_report/views/product_product_views.xml [hunk 1]
```diff
@@ -38,6 +39,7 @@
                     name="action_view_amls"
                     type="object"
                     icon="fa-info-circle"
+                    title="View AMLS"
```
Comment (rousseldenis) [inline stock_account_valuation_report/views/product_product_views.xml]: @RicardCForgeFlow I'd prefer explicit wording (View Journal Items).

**Review by rousseldenis — COMMENTED** [review]
Code review. Little change

File: stock_account_valuation_report/views/product_product_views.xml (hunk 1 above)
Comment (RicardCForgeFlow) [reply stock_account_valuation_report/views/product_product_views.xml]: Okay, thanks for the review :)

**Review by CasVissers-360ERP — APPROVED** [review]
Functional review

**Comment by CasVissers-360ERP** [comment]
Can this be merged?

**Review by hparfr — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**